Updating Okta IDP configuration (#10470)
DataDog/documentation: content/en/account_management/saml/okta.md

## General details

|Okta IDP Input Field   | Expected Value |
|---|---|

> | Single Sign On URL  | Assertion Consumer Service URL (Find this URL on the [Configure SAML][1] page, in the `Assertion Consumer Service URL` field.)  |

Reviewer: ```suggestion
| Single Sign On URL  | Assertion Consumer Service URL (Find this URL on the [Configure SAML][1] page, in the *Assertion Consumer Service URL* field.)  |
```

> | Recipient URL   | Assertion Consumer Service URL (or click checkbox: `Use this for Recipient URL and Destination URL`)  |
> | Destination URL   | Assertion Consumer Service URL (or click checkbox: `Use this for Recipient URL and Destination URL)`  |

Reviewer: ```suggestion
| Recipient URL   | Assertion Consumer Service URL (or click the *Use this for Recipient URL and Destination URL* checkbox)  |
| Destination URL   | Assertion Consumer Service URL (or click the *Use this for Recipient URL and Destination URL* checkbox)  |
```

> | Audience URI (SP Entity ID)   | Service Provider Entity ID (Find this URL on the [Configure SAML][1] page, in the `Service Provider Entity ID` field.) |

Reviewer: ```suggestion
| Audience URI (SP Entity ID)   | Service Provider Entity ID (Find this URL on the [Configure SAML][1] page, in the *Service Provider Entity ID* field.) |
```

| Name ID Format  | EmailAddress |
| Response | Signed  |
| Assertion Signature | Signed |
| Signature Algorithm  | SHA256   |
| Assertion Encryption  | Assertions can be encrypted, but unencrypted assertions are also accepted.  |
| SAML Single Logout   | Disabled  |
| authnContextClassRef  | PasswordProtectedTransport |
| Honor Force Authentication  | Yes  |

> | SAML Issuer ID  | http://www.okta.com/${org.externalKey}  |

Reviewer: ```suggestion
| SAML Issuer ID  | `http://www.okta.com/${org.externalKey}`  |
```

## Attribute statements details

| Name  |Name Format (optional)   | Value  |
|---|---|---|

> | NameFormat   | URI Reference | urn:oasis:names:tc:SAML:2.0:attrname-format:uri  |
> | sn  | URI Reference  | user.lastName |
> | givenName  | URI Reference  | user.firstName  |

Reviewer: ```suggestion
| NameFormat   | URI Reference | `urn:oasis:names:tc:SAML:2.0:attrname-format:uri`  |
| sn  | URI Reference  | `user.lastName` |
| givenName  | URI Reference  | `user.firstName`  |
```

## Group attribute statements (optional)

> This is only required if you are leveraging [Authn Mapping][4].

Reviewer: ```suggestion
This is required only if you are using [AuthN Mapping][4].
```

| Name  |Name Format (optional)   | Value  |
|---|---|---|

> | memberOf   | Unspecified| Matches regex: .* (This method retrieves all groups. Please reach out to your IDP administrator if this does not fit your use case.)  |

Reviewer: ```suggestion
| memberOf   | Unspecified| Matches regex `.*` (This method retrieves all groups. Contact your IDP administrator if this does not fit your use case.)  |
```

[1]: https://app.datadoghq.com/saml/saml_setup
[2]: /account_management/saml/
[3]: https://support.okta.com/help/s/article/How-do-we-download-the-IDP-XML-metadata-file-from-a-SAML-Template-App

> [4]: https://docs.datadoghq.com/account_management/saml/#mapping-saml-attributes-to-datadog-roles

Reviewer: ```suggestion
[4]: /account_management/saml/#mapping-saml-attributes-to-datadog-roles
```
